### PR TITLE
bump elixir version to 1.9.0

### PR DESCRIPTION
**Pull request by ayrat555**
https://elixir-lang.org/blog/2019/06/24/elixir-v1-9-0-released/

## Changelog

- bump elixir version to 1.9.0

### CODE REVIEW

**Comment by ayrat555** [comment]
@vbaranov done. pr is ready

File: apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/rolling_window.ex [hunk 1]
```diff
@@ -211,12 +211,16 @@ defmodule EthereumJSONRPC.RollingWindow do
...
-    case :ets.lookup(table, key) do
-      [{_, current_window, windows}] ->
-        [current_window | windows]
-
-      _ ->
-        []
+    if :ets.whereis(table) == :undefined do
+      []
+    else
+      case :ets.lookup(table, key) do
```
Comment (pasqu4le) [inline apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/rolling_window.ex]: Small nitpick: `:ets.lookup` works as well with the table `tid`, but avoids a name resolution.

You could take advantage of this, by changing the `if :ets.whereis(table)` in a `case`

Comment (ayrat555) [reply apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/rolling_window.ex]: Can you please provide an example.
For not existing table
```
:ets.lookup(:table, "key")
** (ArgumentError) argument error
    (stdlib) :ets.lookup(:table, "key")
```

Comment (pasqu4le) [reply apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/rolling_window.ex]: No that's right, what I meant was to change the outer `if` block, including the `whereis`, to be a `case`:
```elixir
    case :ets.whereis(table) == :undefined do
      :undefined -> []
      tid ->
        case :ets.lookup(tid, key) do
          [{_, current_window, windows}] ->
            [current_window | windows]

          _ ->
            []
        end
    end
```
so to use the `tid` that `whereis` provides anyway

Comment (pasqu4le) [reply apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/rolling_window.ex]: Also, I just noticed that the typespec returns a `nonempty_list`, that does not seem right, could you change that?

Comment (ayrat555) [reply apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/rolling_window.ex]: done

**Pull request merged**